README.md

```diff
@@ -481,6 +481,45 @@ default toolchain:
 rustup default system
 ```
 
+## Working with components
+
+Components that enhance your default Rust toolchain, like `rustfmt`, can be
+installed like so:
+
+```
+rustup component add rustfmt
+```
+
+This will install `rustfmt` for the toolchain you have selected as your default
+using the `rustup toolchain default <toolchain>` command. If you want to see a list of the
```

Comment:
I'd split these two sentences and put, in between them, something like "You can use the \`--toolchain\` argument to add a component to a specific toolchain."